src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor.cs

```diff
@@ -262,9 +262,16 @@ private void NavigateTo(string? applicationId, LogLevel? level)
         NavigationManager.NavigateTo(url);
     }
 
-    private static string GetRowClass(OtlpLogEntry entry)
+    private string GetRowClass(OtlpLogEntry entry)
     {
-        return $"log-row-{entry.Severity.ToString().ToLowerInvariant()}";
+        if (entry == _selectedLogEntry)
+        {
+            return "selected-row";
+        }
+        else
+        {
+            return $"log-row-{entry.Severity.ToString().ToLowerInvariant()}";
```

Comment:
What is combination of log-row-* & selected-row? Could be weird format change on row when selecting-deselecting depending on severity.

Reply:
There is no combination. It looks like selected row takes precedence. It's not ideal, but I want to try it out before passing judgement.

Reply:
Yeah having selected override the severity was kind of picking the option I disliked the least. Severity overriding selection seemed worse, and trying to come up with a color combo for severity + selection didn't seem likely to work out. I debated outlining + background, but figured something like that should come from feedback.

Reply:
Tried it out and it is Good Enough. The error/warning icons are still visible.

Coming up with color combos for every situation in light and dark would be a lot of work for not much value.